doc/reference/modules/nhibernate_caches.xml

```diff
@@ -568,9 +568,9 @@
 	</section>
 
 	<section id="NHibernate.Caches.RtMemoryCache">
-		<title>Runtime Memory Configuration</title>
+		<title>RtMemoryCache Configuration</title>
 		<para>
-			As runtime memory relies on <classname>System.Runtime.Caching.MemoryCache</classname> for the
+			RtMemoryCache relies on <classname>System.Runtime.Caching.MemoryCache</classname> for the
 			underlying implementation.
 			Following NHibernate configuration settings are available:
```

Comment:
The following

Reply:
I frequently read articles where I find similar article omissions. Maybe I am missing the rule allowing their omission, or maybe that is an american thing (I am reading mainly an american paper, IEEE Spectrum). I would be glad if you can tell me more about that.